acasclient/lsthing.py

```diff
@@ -2028,7 +2028,7 @@ def get_file_hash(self, file_path):
                 buf = file_ref.read(BLOCKSIZE)
         return(hasher.hexdigest())
 
-    def add_link(self, verb=None, linked_thing=None, recorded_by=None, metadata=None, results=None):
+    def add_link(self, verb=None, linked_thing=None, recorded_by=None, metadata=None, results=None, subject_type=None, **kwargs):
```

Comment:
add subject_type as param, and **kwargs to pass to SimpleLink. This allows us to override subject_type and object_type in SImpleLink

Reply:
Is there a reason why `object_type` is buried in kwargs but `subject_type` is exposed directly?